source/unified-test-format/schema-1.3.json

```diff
@@ -81,7 +87,22 @@
               "minItems": 1,
               "items": {
                 "type": "string",
-                "enum": ["commandStartedEvent", "commandSucceededEvent", "commandFailedEvent"]
+                "enum": [
+                  "commandStartedEvent",
+                  "commandSucceededEvent",
+                  "commandFailedEvent",
+                  "poolCreatedEvent",
+                  "poolReadyEvent",
+                  "poolClearedEvent",
+                  "poolClosedEvent",
+                  "connectionCreatedEvent",
+                  "connectionReadyEvent",
+                  "connectionClosedEvent",
+                  "connectionCheckOutStartedEvent",
+                  "connectionCheckOutFailedEvent",
+                  "connectionCheckedOutEvent",
+                  "connectionCheckedInEvent"
```

Comment:
The reason why I used capitalized names was because this is how the events were defined in the specs and how they were defined in the code (class names are capitalized). Hence it seemed to me that beginning with lower case would 1) require transformations throughout the test runner to convert back and forth to the actual event names used, and 2) would raise the issue of the names in tests not matching exactly the names defined in specifications.

What was the rationale for using lowercased names? Perhaps I am missing something.

Reply:
There was no rationale on my end besides the fact that the existing events (`commandStartedEvent`, `commandSucceededEvent`, and `commandFailedEvent`) are lowercased.

Reply:
I think we talked about this, but the original rationale for the unified test runner was that the names from the older test format were `snake_case` (see [CRUD v2](https://github.com/mongodb/specifications/tree/master/source/crud/tests#legacy-v2-test-format)). I swapped the underscores for capitalization, but left the first letter as-is.

Reply:
In my opinion  starting with capital letter would be consistent with both specs and driver code for events. Starting with a small letter might be consistent with other (non-event) values. I think the first consistency is more important in this case but perhaps others disagree.